src/Pkg.jl

```diff
@@ -126,7 +126,10 @@ const add = API.add
 """
     Pkg.precompile()
 
-Precompile all the dependencies of the project.
+Precompile all dependencies of the project via a depth-first depdendency tree-based 
+parallel precompilation of all dependencies in the project manifest. 
```

Comment:
Perhaps this is too detailed?
```suggestion
Precompile all the dependencies of the project in parallel.
```